clang/test/CodeGenSYCL/intel-fpga-local.cpp

```diff
@@ -25,24 +27,17 @@
 
 // CHECK-BOTH: @llvm.global.annotations
 // CHECK-DEVICE-SAME: { i8* addrspacecast (i8 addrspace(1)* bitcast (i32 addrspace(1)* @_ZZ15attrs_on_staticvE15static_numbanks to i8 addrspace(1)*) to i8*)
-// CHECK-DEVICE-SAME: [[ANN_numbanks_4]]{{.*}}i32 38
-// CHECK-DEVICE-SAME: { i8* addrspacecast (i8 addrspace(1)* bitcast (i32 addrspace(1)* @_ZZ15attrs_on_staticvE15static_annotate to i8 addrspace(1)*) to i8*)
+// CHECK-DEVICE-SAME: [[ANN_numbanks_4]]{{.*}}, i32 39
+// CHECK-DEVICE-SAME: { i8* addrspacecast (i8 addrspace(1)* bitcast (i32 addrspace(1)* @_ZZ15attrs_on_staticvE15static_annotate to i8 addrspace(1)*) to i8*) 
 // CHECK-HOST-SAME: { i8* bitcast (i32* @_ZZ15attrs_on_staticvE15static_annotate to i8*)
-// CHECK-BOTH-SAME: [[ANN_annotate]]{{.*}}i32 42
+// CHECK-BOTH-SAME: [[ANN_annotate]]{{.*}} i32 40
```

Comment:
```suggestion
// CHECK-BOTH-SAME: [[ANN_annotate]]{{.*}}i32 40
```

Reply:
Could you clarify the value of this change, please?

Reply:
No need to add extra symbols to source code

Reply:
Victoria explained me, that it is point in string number of static_annotate declaration.

Reply:
I think it might useful to make sure there is a space before `i32` type as it limits the scope of potential matches and makes it more reliable (?). Am I too paranoid?

Reply:
I don't think that it is matter.

Reply:
Okay, it can be useful to leave a space. If you will do so, please do the same on line 33

Reply:
Okey.